### PR TITLE
Fix DNS Dig lookups for tests

### LINKED ISSUE #467: DNS:Dig failures in tests
Service seems to be unresponsive -- has been for a week. Specifically, the Wikipedia lookup service.

### PR DESCRIPTION
**Pull request by fhcarter**
Fixes #467

Fix DNS Dig lookups, specifically the wikipedia lookups.  Change relevant tests to do simple domain lookup.

### CODE REVIEW

File: camelConnector/src/test/java/io/vantiq/extsrc/camelconn/discover/VantiqComponentResolverTest.java [hunk 1]
```diff
@@ -434,20 +435,27 @@ public void testHdrDupSetup() {
...
     
     
-    public static final String QUERY_MONKEY = "monkey.wp.dg.cx";
-    public static final String RESPONSE_MONKEY = "\"A Macaque, an old world species of "
-            + "monkey native to Southeast Asia|thumb]A monkey is a primate of the "
-            + "Haplorrhini suborder and simian infraorder, either an Old World monkey "
-            + "or a New World monkey, but excluding apes. There are about 260 known "
-            + "living specie\" \"s of monkey. Many are arboreal, although there are "
-            + "species that live primarily on the ground, such as baboons... "
-            + "http://en.wikipedia.org/wiki/Monkey\"";
```
Comment (fhcarter) [inline camelConnector/src/test/java/io/vantiq/extsrc/camelconn/discover/VantiqComponentResolverTest.java:444]: Leaving these around for when service comes back online

File: camelConnector/src/test/java/io/vantiq/extsrc/camelconn/discover/VantiqComponentResolverTest.java [hunk 2]
```diff
@@ -731,6 +754,8 @@ public static TriFunction<CamelContext, String, Object, Boolean> defineVerifyOpe
                         if (rawData instanceof Map){
                             //noinspection unchecked,rawtypes
                             msgMap = (Map) rawData;
```
Comment (fhcarter) [inline camelConnector/src/test/java/io/vantiq/extsrc/camelconn/discover/VantiqComponentResolverTest.java:756]: little bit of cleanup as we were getting unexpected errors due to unrelated change

**Comment by fhcarter** [comment]
Only change is in tests -- no rebuild or republish necessary

**Comment by fhcarter** [comment]
[Jenkins run](https://jenkins.vantiq.com/jenkins/job/vantiq-extension-sources-branch/224/)

**Review by jcguion — APPROVED** [review]
LGTM

**Comment by fhcarter** [comment]
@rblumer -- extsrc tests should be better now...

**Pull request merged**